agents-api/agents_api/autogen/openapi_model.py

```diff
@@ -523,7 +513,9 @@ class ChatSettings(BaseModel):
     **Important:** when using JSON mode, you **must** also instruct the model to produce JSON yourself via a system or user message. Without this, the model may generate an unending stream of whitespace until the generation reaches the token limit, resulting in a long-running and seemingly "stuck" request. Also note that the message content may be partially cut off if `finish_reason="length"`, which indicates the generation exceeded `max_tokens` or the conversation exceeded the max context length.
 
     """
-    seed: Annotated[int | None, Field(None, ge=-1, le=9999)]
```

Comment:
this was rolled back incorrectly. needs to be small, otherwise the js schema generator freaks out